proto/backend.proto

```diff
@@ -0,0 +1,212 @@
+syntax = "proto3";
+import "google/protobuf/empty.proto";
+package grafana.plugin;
```

Comment:
I think maybe this package is something like grafana.backend.plugin (since this whole package is for backend plugins) , and then the `BackendPlugin` service can be renamed.